portal/forms/home.py

```diff
@@ -65,7 +64,14 @@ def clean_name(self):
 
     def clean_message(self):
         message = self.cleaned_data.get("message", None)
-        if re.match(re.compile('^[ -~]+$'), message) is None:
+        line_list = message.splitlines()
+        raise_error = False
+
+        for line in line_list:
+            if re.match(re.compile('^[ -~]+$'), line) is None:
+                raise_error = True
```

Comment:
that's a bit inefficient to wait until the for finishes to raise the error, as if the invalid character appears in the first line you still have to read all the lines here... I'd rather raise directly there rather than use the boolean